tools/fastq_provider/fastq_provider.xml

```diff
@@ -54,28 +92,58 @@
 	    </param>
         <param name="library" argument="-l" type="text"  label="ENA library (ENA source files only), by default inferred from file name" help="library, by default inferred from file name. E.g. -l ERR1888646 " optional='true'/>
         <param name="config_file" argument="-c" type="data" format="atlas-fastq-provider-config.sh" label="config file to override defaults" help="config file to override defaults" optional='true'/>
-        <param name="validate_only" argument="-v" type="boolean" label="validate only to check a source URI is valid, don't download" help="validate only, don't download" optional='true'/>
-        <param name="download_type" argument="-d" type="select" label="download type, fastq or SRA file" help="download type, fastq or srr" optional='true'>
-            <option value="fastq" selected="true">fastq</option>
-            <option value="srr">SRA</option>
-	    </param>
+
     </inputs>
+
     <outputs>
-        <data name="outfile" label="${tool.name} on ${on_string}: compressed fastq file" format="fastq.gz"  />
+        <conditional name="main_source">
```

Comment:
Outputs, AFAIK, don't use conditionals but filters based on the conditionals, that is why the lintern is failing:

```
.. ERROR: Invalid XML found in file: tmpvitalx03. Errors [/tmp/tmpvitalx03:96:0:ERROR:SCHEMASV:SCHEMAV_ELEMENT_CONTENT: Element 'conditional': This element is not expected. Expected is one of ( output, data, collection ).]
```

Reply:
See this part of the docs https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-outputs-data-filter

Reply:
Done in 85a751e490f70cae8a1e67c826c1441572c3a2b4